src/Umbraco.Web.UI.Client/src/views/dashboard/dashboard.tabs.controller.js

```diff
@@ -48,7 +48,7 @@ function startUpDynamicContentController($q, $timeout, $scope, dashboardResource
             {
                 title: "Documentation",
                 description: "Find the answers to your Umbraco questions",
-                url: "https://our.umbraco.com/documentation/?utm_source=core&utm_medium=dashboard&utm_content=text&utm_campaign=documentation/"
+                url: "https://docs.umbraco.com/?utm_source=core&utm_medium=dashboard&utm_content=text&utm_campaign=documentation/"
```

Comment:
Not sure if I updated this link correctly, as it looks like it's linked with some tracking as well?

Reply:
Just had a look at it, and I have to be honest I'm not sure if that link is actually used.

The content of the dashboard (including the links) is actually not managed by the CMS or core team, it's instead fetched from .com  (`https://dashboard.umbraco.com/`, as a grid no less 😱), so this doesn't seem to change the dashboard. 

But regardless, I think this change is fine to merge, but to update the link in the dashboard I think you need to get a hold of COMA unfortunately 😄

Reply:
Have already booked COMA peeps to talk about this, so we should be covered on this front 💪 
But thanks for highlighting it!